### ISSUE TITLE
Android solution

### ISSUE THREAD

**Issue by IDKwhattoputhere** [reek/anti-adblock-killer#1215]
Tampermonkey has been available for Android for some time but it's not viable, especially since AakList can't be used with it.

That's why I wanted to let you know about this solution which makes Aak usable on Android: 
**[usi (User|Unified Script Injector)](https://addons.mozilla.org/android/addon/userunified-script-injector/)**

Not very extensive yet but I added AakScript in the settings, tried it on the Antiblock.org V3 Demo and it worked in combination with uBlock Origin while uBlock Origin alone failed.

**Comment by ciborge** [comment]
Issue is not the place for this kind of share but thanks :+1:

**Comment by IDKwhattoputhere** [comment]
I know it isn't but this is the one place where most people would see it.

**Comment by IDKwhattoputhere** [comment]
I know it isn't but this is the one place where most people would see it.

**Comment by reek** [comment]
@IDKwhattoputhere 
you did well. I'll try that.

**Comment by reek** [comment]
@IDKwhattoputhere 
I have created a wiki https://github.com/reek/anti-adblock-killer/wiki/Mobile

Let me know if you find other ways to use Aak :smile:

**Comment by TPS** [comment]
@reek Since https://github.com/AdguardTeam/AdguardForAndroid/issues/420 has landed, maybe you want to add that somewhere on https://github.com/reek/anti-adblock-killer/wiki/Mobile, & then maybe update is again after https://github.com/AdguardTeam/AdguardForAndroid/issues/34 lands.

**Comment by reek** [comment]
@TPS 
Yes, I plan to add instructions for AdGuard, so I'll leave the issue open
